README.md

```diff
@@ -68,19 +77,23 @@ Finally, scroll to the end of the page and hit **Save**.
 Whenever you enable the `spotify-remote-mode` minor mode you get the following
 key bindings:
 
-| Key                | Function                     | Description                       |
-|:-------------------|:-----------------------------|:----------------------------------|
-| <kbd>M-p M-s</kbd> | `spotify-toggle-shuffle`     | Turn shuffle on/off [1]           |
-| <kbd>M-p M-r</kbd> | `spotify-toggle-repeat`      | Turn repeat on/off [1]            |
-| <kbd>M-p M-p</kbd> | `spotify-toggle-play`        | Play/pause                        |
-| <kbd>M-p M-f</kbd> | `spotify-next-track`         | Next track                        |
-| <kbd>M-p M-b</kbd> | `spotify-previous-track`     | Previous track                    |
-| <kbd>M-p p m</kbd> | `spotify-my-playlists`       | Show your playlists               |
-| <kbd>M-p p f</kbd> | `spotify-featured-playlists` | Show the featured playlists       |
-| <kbd>M-p p s</kbd> | `spotify-playlist-search`    | Search for playlists              |
-| <kbd>M-p p u</kbd> | `spotify-user-playlists`     | Show playlists for the given user |
-| <kbd>M-p p c</kbd> | `spotify-create-playlist`    | Create a new playlist             |
-| <kbd>M-p t s</kbd> | `spotify-track-search`       | Search for tracks                 |
+| Key                | Function                     | Description                                     |
+|:-------------------|:-----------------------------|:------------------------------------------------|
+| <kbd>M-p M-s</kbd> | `spotify-toggle-shuffle`     | Turn shuffle on/off [1]                         |
+| <kbd>M-p M-r</kbd> | `spotify-toggle-repeat`      | Turn repeat on/off [1]                          |
+| <kbd>M-p M-p</kbd> | `spotify-toggle-play`        | Play/pause                                      |
+| <kbd>M-p M-f</kbd> | `spotify-next-track`         | Next track                                      |
+| <kbd>M-p M-b</kbd> | `spotify-previous-track`     | Previous track                                  |
+| <kbd>M-p p m</kbd> | `spotify-my-playlists`       | Show your playlists                             |
+| <kbd>M-p p f</kbd> | `spotify-featured-playlists` | Show the featured playlists                     |
+| <kbd>M-p p s</kbd> | `spotify-playlist-search`    | Search for playlists                            |
+| <kbd>M-p p u</kbd> | `spotify-user-playlists`     | Show playlists for the given user               |
+| <kbd>M-p p c</kbd> | `spotify-create-playlist`    | Create a new playlist                           |
+| <kbd>M-p t s</kbd> | `spotify-track-search`       | Search for tracks                               |
+| <kbd>M-p v u</kbd> | `spotify-volume-up`          | Increase the volume [2]                         |
+| <kbd>M-p v d</kbd> | `spotify-volume-down`        | Decrease the volume [2]                         |
+| <kbd>M-p M-d</kbd> | `spotify-select-device`      | Select a playback device [2]                    |
+
```

Comment:
I suggest some changes in shortcuts.

All key binding that manipulate the player state follows the convention `M-p M-<key>`, so I suggest using that prefix to volume up/down commands, and change the device selection shortcut to `M-p d`.

What do you think?